Add chapter about our interview process. (#4)
someengineering/company-handbook: interview-process.md

# Interview Process

> If you are interested in a job at Some Engineering Inc., please send your application to [hr@some.engineering](mailto:hr@some.engineering).

Reviewer: ```suggestion
If you are interested in a job at Some Engineering Inc., please email your resume/CV to [hr@some.engineering](mailto:hr@some.engineering).
```

> ## Interview process for engineers
> 
> The interview process for engineers usually has those steps:

Reviewer: ```suggestion
## Engineering Interview Process

For engineering candidates, there are four stages to the interview process.
```

> ### Initial Contact

Reviewer 2: Intro Call
